docs/src/src/interfaces/IPolygonEcosystemToken.sol/interface.IPolygonEcosystemToken.md

```diff
@@ -115,7 +115,7 @@ function PERMIT2() external view returns (address);
 
 ### mintPerSecondCap
 
-*13.37 POL tokens per second. will limit emission in ~23 years*
+*13.37 POL tokens per second. will limit emission in ~12 years*
```

Comment:
Better to confirm that everyone expects this numbers. Can you share maths to double check such 12 years from 23 years? Is it due to 2 to 3% increase?

Reply:
The math is checking when the emission rate per second and looking after how many years it exceeds the threshold of the mintcap. It's just about 12 years: https://www.wolframalpha.com/input?i=10000000000*(1.03%5E(x+1/(365*24*60*60))-1.03%5Ex)+%3D+13.37